oxidauth-postgres/migrations/20240416170202_create_auth_keys.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE auth_keys (
+    id UUID PRIMARY KEY NOT NULL DEFAULT uuid_generate_v4(),
+    user_id UUID FOREIGN KEY REFERENCES Users(id) NOT NULL,
+    code VARCHAR(256) UNIQUE NOT NULL,
```

Comment:
Code could be the primary key, not sure having the extra id column here helps with anything